app/views/microposts/_micropost.html.erb

```diff
@@ -0,0 +1,14 @@
+<li id="micropost-<%= micropost.id %>">
+  <%= link_to gravatar_for(micropost.user, size: Settings.gravatar_size), micropost.user %>
+  <span class="user"><%= link_to micropost.user.name, micropost.user %></span>
+  <span class="content">
+    <%= micropost.content %>
+    <%= image_tag micropost.picture.url if micropost.picture? %>
```

Comment:
you should research `how to set a default image in carierwave gem`, so that you do not have to check the picture exists or not like this